workbench-core/accounts/src/constants/projectStatus.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+// https://www.damirscorner.com/blog/posts/20200619-StringLiteralTypeGuardInTypescript.html
+// This file is structured this way so we can verify whether a user input is of type `ProjectStatus`
+export const PROJECT_STATUS: string[] = ['AVAILABLE', 'SUSPENDED', 'DELETED'];
```

Comment:
Kevin was discussing this. We might revisit for all status constants. I think there was an important check somewhere that turned us on to this solution. `EnvironmentStatus` was the OG so the implemented for that would know best the implementation reason.

Reply:
It was implemented [here](https://github.com/aws-solutions/solution-spark-on-aws/blob/6269b342e6d8a9fcec7c8c65c25d155d00b00c59/solutions/swb-app/src/environmentRoutes.ts#L240) to check the status for the request is correct. I think in this particular case, it's not needed now because we don't actually do that check since STATUS is not a part of the request. 

However, maybe in the future we will allow user to mark a project status. I'm ok with keeping this as is for now.